introduction.rst

```diff
@@ -1,3 +1,225 @@
+* Gelly is a graph processing API and library
+* Flink CEP is a "complex event processing" API that allows to detect and react to occurrences of event patterns in a stream
+
+Motivation
+~~~~~~~~~~
+
+* Many applications require cyclic streams (parallel model training, evaluation)
+* Many applications require real-time true streaming
+
+Features
+~~~~~~~~
+
+* High performance, low latency
+* Support for event time and out-of-order events
+* Exactly-once semantics for stateful computations
+* Highly flexible streaming windows
+* Continuous streaming model with backpressure
+* Fault tolerance via lightweight distributed snapshots
+* A unified runtime systems for batch and stream processing
+* Built-in memory management
+* Native support for iterations and delta iterations
+* Lazy DAG evaluation with a sophisticated optimization engine
+* Rich library ecosystem
+* Broad integration with other software products in the "Big Data"/Hadoop ecosystem
+
+Alluxio
+-------
+
+`Alluxio <http://alluxio.org/>`_ is a memory-speed virtual distributed storage system.
+
+Motivation
+~~~~~~~~~~
+
+* Need to leverage memory processing
+* Need to share context among different jobs
+* Need for fault tolerance among different jobs
+
+Features
+~~~~~~~~
+
+* Context sharing across (heterogeneous) jobs
+* Lineage: reliable file sharing without replication
+
+  * One copy of data in memory (fast)
+  * Upon failure, re-compute data using lineage (fault tolerant)
+
+* Flexible tiered storage: leverage memory, SSDs or HDDs according to your needs (fine tune speed and capacity requirements)
+
+  * Pluggable data management policy (promote hot data to upper tiers, evict cold data to lower tiers)
+  * Transparent naming across several distributed filesystems
+
+* Unified namespace
+
+  * Transparent naming across several distributed file systems
+  * Shared data across heterogeneous storage systems
+  * Mount/unmount on-the-fly
+
+Cassandra
+---------
+
+`Apache Cassandra <https://cassandra.apache.org>`_ is an Open Source distributed database management system designed to handle large amounts of data across many commodity servers, providing high availability with no single point of failure.
+
+Motivation
+~~~~~~~~~~
+
+* Need to integrate a highly available and performing column oriented data store to support a more flexible data model
+* Need to integrate a store, which could be used for real-time and time series analytics
+
+Features
+~~~~~~~~
+
+* Peer-to-peer architecture, highly available design with no single point of failure
+* No shared storage requirements for failover
+* Simple deployment and maintenance
+* Multi-datacenter support
+* Hadoop-compatible
+* File-append support
+* Multiple file systems
+* Tuneable data locality
+* Small files optimization
+* Per-file segment and block size support
+* Sequential and random access to file support
+* Security: native authentication, Kerberos support
+* File watcher notification service
```

Comment:
Where did you take this feature? Is it in the official documentation of Cassandra?